alecs-theme/src/WEB-INF/src/resources-importer/journal/templates/Card Row/Card Row.ftl

```diff
@@ -0,0 +1,32 @@
+<style>
+    .autofit-col {
+        display: inline-block;
+    }
+    .icon-shopping-cart {
+        margin-left: auto;
+        font-size: 32px;
+        color: #000;
+    }
+    .icon-shopping-cart:hover {
+        cursor: pointer;
+        text-decoration: none;
+        color: #323232;
+    }
+    .card {
+        transition: 1s;
+        cursor: pointer;
+    }
+    .card:hover {
+        transform: scale(1.05);
+        box-shadow: 0 1px 20px -4px rgba(0, 0, 0, 0.6);
+    }
+</style>
+<div class="row">
+    <#if HTML18zr.getSiblings()?has_content>
```

Comment:
We should change this so the ```<div class="row />``` is wrapped within the ```<#if />``` check since if there is no content, then this would just output a blank ```<div class="row" />```.